Day3/exercise2/Controllers/Student.go

```diff
@@ -0,0 +1,72 @@
+package Controllers
+
+import (
+	"fmt"
+	"freshers-bootcamp/Day3/exercise2/Models"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func GetAllRecords(context *gin.Context){
+	var records [] Models.Student
+	fmt.Println(records)
+
+	err := Models.GetAllRecords(&records)
+
+	if err != nil {
+		context.AbortWithStatus(http.StatusNotFound)
+	} else {
+		context.JSON(http.StatusOK, records)
+	}
+}
+
+func CreateStudent(context *gin.Context){
+	var record Models.Student
+	context.BindJSON(&record)
+	err := Models.AddRecords(&record)
+
+	if err != nil {
+		fmt.Println(err.Error())
+		context.AbortWithStatus(http.StatusNotFound)
```

Comment:
error status should be different, should be according to error, not a generic one